src/Install/Installer.php

```diff
@@ -299,12 +299,16 @@ private function installSaferPayCardAlias()
     private function installSaferPayLog()
     {
         return Db::getInstance()->execute(
-            'CREATE TABLE IF NOT EXISTS ' . _DB_PREFIX_ . 'saferpay_log' . '(
-            `id_saferpay_log` INTEGER(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-            `message` TEXT NOT NULL,
-            `payload` TEXT NOT NULL,
-            `date_add` datetime NOT NULL
-                ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci'
+            'CREATE TABLE IF NOT EXISTS ' . _DB_PREFIX_ . pSQL(\SaferPayLog::$definition['table']) . '(
+                `id_saferpay_log` INTEGER(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+                `id_log` INTEGER(10) DEFAULT 0,
+                `id_shop` INTEGER(10) DEFAULT 1,
+                `message` TEXT NOT NULL DEFAULT "",
+                `request` TEXT NOT NULL DEFAULT "",
+                `response` TEXT NOT NULL DEFAULT "",
+                `context` TEXT NOT NULL DEFAULT "",
+                `date_add` datetime NOT NULL DEFAULT NOW()
+                    ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci'
```

Comment:
We might want to add index as with debug mode it could collect lots of logs which might slow down the page 
`            INDEX(`id_log`)
`